src/views/liste-aides.vue

```diff
@@ -83,9 +83,11 @@ import institutionsBenefits from "generator:institutions"
 import CommuneMethods from "@/lib/commune.js"
 import { Commune } from "@lib/types/commune.d.js"
 import { capitalize } from "@lib/utils.js"
+import { Benefit } from "@data/types/benefits"
 
 const zipCode = ref("")
 const selectedCommune = ref<Commune | null>()
+const selectedBenefits = ref<Benefit | null>()
```

Comment:
Suggestion de nom pour cette variable qui semble concerner les mots-clés 

```suggestion
const searchTerms = ref<string | null>()
```